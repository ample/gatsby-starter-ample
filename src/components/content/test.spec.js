import React from "react"
import renderer from "react-test-renderer"

import Component from "./"

describe("Component", () => {
  it("renders correctly", () => {
    const tree = renderer.create(<Component body="<p>Hello world</p>" />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
