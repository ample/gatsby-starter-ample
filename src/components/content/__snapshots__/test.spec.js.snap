// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Content renders correctly 1`] = `
<div
  className="content"
  dangerouslySetInnerHTML={
    Object {
      "__html": "<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.</p>",
    }
  }
/>
`;
